Makefile

```diff
@@ -39,12 +39,34 @@ init:
 # Build all Haskell services and executables, run unit tests
 .PHONY: install
 install: init
+ifeq ($(WIRE_BUILD_WITH_CABAL), 1)
+	cabal build all
+	cabal test all
+	cabal install --install-method=copy --installdir=dist all
```

Comment:
`cabal install` builds things even after `cabal build` has built it. I am going to see if I can do something to influence that, otherwise we can maybe just use `cabal-plan list-bins` to copy all the binaries to dist.

Reply:
Resolved by using `cabal-plan` and a couple of tiny shell scripts.